recipe/build.sh

```diff
@@ -4,4 +4,14 @@ set -eux
 mkdir -p $PREFIX/etc/bash_completion.d;
 cp etc/hub.bash_completion.sh $PREFIX/etc/bash_completion.d/hub.sh
 
-go install -v .
+# go install -v .
+
+if [[ "$target_platform" = "osx-arm64" ]]; then  
+  go build -v .
+  mkdir -p ${PREFIX}/bin
+  install hub ${PREFIX}/bin
```

Comment:
are these all files that need to be installed?

Reply:
I believe so, but let me double check locally first.

Reply:
Yes, all good.